ui/src/app/views/admin-review-workspace/component.spec.ts

```diff
@@ -0,0 +1,84 @@
+import {ComponentFixture, fakeAsync, TestBed, tick} from '@angular/core/testing';
+import {FormsModule} from '@angular/forms';
+import {RouterTestingModule} from '@angular/router/testing';
+
+import {ClarityModule} from '@clr/angular';
+
+import {
+  BugReportService,
+  ProfileService,
+  WorkspacesService
+} from 'generated';
+
+import {BugReportServiceStub} from 'testing/stubs/bug-report-service-stub';
+import {ProfileServiceStub} from 'testing/stubs/profile-service-stub';
+import {ProfileStorageServiceStub} from 'testing/stubs/profile-storage-service-stub';
+import {ServerConfigServiceStub} from 'testing/stubs/server-config-service-stub';
+import {WorkspacesServiceStub} from 'testing/stubs/workspace-service-stub';
+
+import {
+  updateAndTick
+} from '../../../testing/test-helpers';
+
+import {ProfileStorageService} from '../../services/profile-storage.service';
+import {ServerConfigService} from '../../services/server-config.service';
+
+import {AdminReviewWorkspaceComponent} from '../admin-review-workspace/component';
+import {BugReportComponent} from '../bug-report/component';
+import {RoutingSpinnerComponent} from '../routing-spinner/component';
+
+class AdminReviewWorkspacePage {
```

Comment:
It's probably worth having another discussion about the pros/cons of this approach - I know we discussed this briefly a while back, but a comparison of the two approaches with the current team might be useful (and if there's consensus, we should switch existing tests to all match this model).

Reply:
The page approach, you mean?

Reply:
Yes.